Compiler/NFFrontEnd/NFTypeCheck.mo

```diff
@@ -1448,8 +1448,8 @@ function matchTypes
   input Type expectedType;
   input output Expression expression;
   input Boolean allowUnknown = false;
-        output Type compatibleType;
-        output Boolean compatible = true;
+  output Type compatibleType;
```

Comment:
I see! I will restore the whitespaces.

However, I think it would be better to emphasize the outputs using the following formatting:
```
function matchTypes
  input        Type actualType;
  input        Type expectedType;
  input output Expression expression;
  input        Boolean allowUnknown = false;
        output Type compatibleType;
        output Boolean compatible = true;
protected
…
```